docs/index.html

```diff
@@ -1 +1,35 @@
-Test
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Blog Post</title>
+  </head>
+  <body>
+    <img src="https://images.unsplash.com/photo-1546074177-ffdda98d214f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1567&q=80" alt="Describe what's in the photo">
+    <h1>Blog Post Title h1</h1>
+    <span>dd.mm.yyyy 00:00</span>
+    <p>Lead Paragraph – Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
```

Comment:
I understand why you took lorem ipsum. Keep in mind that if you do it for a German Client it's always better to take a German text. Because the Latin structure is way different than the German and it gives you an other feeling when you then switch to the German text. Germen is having more uppercases and long words. To fake English it's usually not hat bad. I tend to use Text in the Language I usually publish. For example from [here](https://www.blindtextgenerator.com/)

Reply:
Kept lorem ipsum cause this page is in english ;) But thanks, good point! Already for the design one should use real text from the project anyway, when ever possible.